src/components/RoomView/gameLog.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import './roomView.css';
+import GuessesRemaining from './GuessesRemaining';
+
+interface ClueType {
+clueString: string;
+clueNumber: number;
+}
+
+interface GameState {
+gameHistory: (string | ClueType)[];
+status: string;
+}
+
+const GameLog: React.FC = () => {
+const game = useSelector<{ game: GameState }, GameState>(
+(state) => state.game
+);
+const gameHistory = game.gameHistory;
+const gameStatus = game.status;
+
+return (
+<div className='gameLog'>
+<h3>Game Status</h3>
+<div>{gameStatus}</div>
+<GuessesRemaining />
```

Comment:
Just noting here TS complaining, but probably because this comp isn't tsx yet? We can work with this for now I think.